client/src/GuessGame/GuessGame.js

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useState } from 'react';
+import chroma from 'chroma-js';
+import countryList from './countryList';
+import Datamap from './DataMap';
+import './GuessGame.css';
+
+
+//Start the game
+const GuessGame = () => {
+  const maxScore = countryList.length;
+  const [guessCounter, setGuessCounter] = useState(0);
+  const [currCountry, setCurrCountry] = useState({});
+  const [data, setData] = useState({});
+  const [win, setWin] = useState(false);
+
+  useEffect(() => {
+    getRandCountry();
+
+    return () => {};
+  }, []);
+
+  //Get a random integer to grab random country.
+  const getRandInteger = (min, max) => {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  };
+
+  // Using random integer get the random country
+  const getRandCountry = () => {
+    const countryNum = getRandInteger(0, maxScore - 1);
+    setCurrCountry(countryList[countryNum]);
+    console.log('Random country: ', currCountry, countryList[countryNum]);
+  };
+
+  //Get the distance via euclidean maths
+  const distance = (srcLat, srcLong) => {
```

Comment:
I enjoy learning about all the logic and math that goes into building this game.